pages/common/mh_copyright.md

```diff
@@ -0,0 +1,12 @@
+# mh_copyright
+
+> Adjust copyright headers for MATLAB or Octave code.
+> More information: <https://misshit.org>.
+
+- Update the year (range) to include the current year for a specific file:
+
+`mh_copyright --primary-entity="{{entity}}" --update-year {{path/to/file.m}}`
+
+- Update the year (range) to include the current year for all files:
```

Comment:
It is not clear where they are updated. Maybe in the current directory?

Reply:
in-place